### ISSUE TITLE
Incorrect Namespace

### ISSUE THREAD

**Issue by kayladnls** [samvaughton/omnipay-barclays-epdq#22]
Hi, I very much value your work on this gateway, however it shouldn't e using the Omnipay namespace. Can you please update it to use your own namespace?

**Comment by samvaughton** [comment]
I don't understand - many third party packages use the `Omnipay` namespace to start with.

http://omnipay.thephpleague.com/gateways/third-party/

To name a few:

 - https://github.com/agmscode/omnipay-agms
 - https://github.com/coatesap/omnipay-realex
 - https://github.com/coatesap/omnipay-datacash

Changing the namespace will also cause breaking changes, and is unnecessary.

**Comment by coatesap** [comment]
I actually think it's probably a good idea, as it means if you want to try out different/competing drivers side by side, there isn't a namespace collision. Also hypothetically if two people create two different drivers that fulfil different integration methods for the same gateway, you might not be able to use them both if they shared the same namespace...

**Comment by samvaughton** [comment]
Yeah Andy the points are valid - any pull requests I'll accept to change
the namespaces as I don't really maintain it anymore.

The version will have to be increase to indicate breaking changes though.

Cheers, Sam
